src/index.html

```diff
@@ -11,12 +11,96 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo-logo"
+          class="header__icon"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <a
+            href="#"
+            class="nav__link is-active"
+          >
+            <li class="nav__item">Apple</li>
+          </a>
+
+          <a
+            href="#"
+            class="nav__link"
+          >
+            <li class="nav__item">Samsung</li>
+          </a>
+
+          <a
+            href="#"
+            class="nav__link"
+          >
+            <li class="nav__item">Smartphones</li>
+          </a>
+
+          <a
+            href="#"
+            class="nav__link"
+            data-qa="hover"
+          >
+            <li class="nav__item">Laptops & Computers</li>
+          </a>
+
+          <a
+            href="#"
+            class="nav__link"
+          >
+            <li class="nav__item">Gadgets</li>
+          </a>
+
+          <a
+            href="#"
+            class="nav__link"
+          >
+            <li class="nav__item">Tablets</li>
+          </a>
+
+          <a
+            href="#"
+            class="nav__link"
+          >
+            <li class="nav__item">Photo</li>
+          </a>
```

Comment:
```suggestion
          <li
            href="#"
            class="nav__link"
          >
            <a class="nav__item">Photo</a>
          </li>
```
That meant the previous comment
`li` - list item is child if `ul` - unordered list

and `a` - link is just content of each list item